tests/playwright/src/bootc-extension.spec.ts

```diff
@@ -124,9 +126,18 @@ describe('BootC Extension', async () => {
         await playExpect.poll(async () => await imagesPage.waitForImageExists(imageName)).toBeTruthy();
       }, 150000);
 
-      test.skipIf(isLinux).each(['QCOW2', 'AMI', 'RAW', 'VMDK', 'ISO'])(
-        `Building bootable image type: %s`,
-        async type => {
+      describe.skipIf(isLinux).each(['QCOW2', 'AMI', 'RAW', 'VMDK', 'ISO'])('Building images ', async type => {
```

Comment:
I'm not sure if it's possible but shouldn't it be something like

```ts
const platforms = '['QCOW2', 'AMI', 'RAW', 'VMDK'];
if (!buildISOImage) {
   platforms.push('ISO')
}

 describe.skipIf(isLinux).each(platforms)('Building images ', async type => {
```

Reply:
Does that work? I was under the impression that vitest creates the test context before anything else, all the examples for parametrized tests that I've seen have the parameters provided directly to the each() without any previous logic.

Reply:
No idea it's why I emitted some doubts :-)

Reply:
@benoitf I just tried it, it seems that you can send params to the test just using a variable, however you can't manipulate the number of tests, basically it uses whatever value `platforms` has if defined globally or it reads it undefined and does not account for any changes to it after the test context is created.

So yes it seems that all the test contexts are created before any logic is executed.

Reply:
Ok thanks so ignore my comment